docs/deploy-development-rp.md

```diff
@@ -134,6 +134,13 @@
      ```bash
      journalctl _COMM=aro -o json --since "15 min ago" -f | jq -r 'select (.COMPONENT != null and (.COMPONENT | contains("access"))|not) | .MESSAGE'
      ```
+     
+   * Perform AdminUpdate on a dev cluster
+   
+     ```bash
+     export CLUSTER=cluster
+	  curl -X PATCH -k "https://localhost:8443/subscriptions/$AZURE_SUBSCRIPTION_ID/resourceGroups/$RESOURCEGROUP/providers/Microsoft.RedHatOpenShift/openShiftClusters/$CLUSTER?api-version=admin" --header "Content-Type: application/json" -d "{}"
+	  ```
```

Comment:
```suggestion
```bash
export CLUSTER=cluster
curl -X PATCH -k "https://localhost:8443/subscriptions/$AZURE_SUBSCRIPTION_ID/resourceGroups/$RESOURCEGROUP/providers/Microsoft.RedHatOpenShift/openShiftClusters/$CLUSTER?api-version=admin" --header "Content-Type: application/json" -d "{}"
```
```
nit: Pull back the `curl` command (because they are two separate commands) to align correctly with the documentation.